core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -153,19 +204,247 @@ describe('Learner dashboard functionality', function() {
     await libraryPage.findExploration(EXPLORATION_FRACTION);
     await libraryPage.addSelectedExplorationToPlaylist();
     await learnerDashboardPage.get();
-    await learnerDashboardPage.navigateToPlayLaterExplorationSection();
+    await learnerDashboardPage.navigateToCommunityLessonsSection();
     await learnerDashboardPage.expectTitleOfExplorationSummaryTileToMatch(
       EXPLORATION_FRACTION);
     await libraryPage.get();
     await libraryPage.findExploration(EXPLORATION_SINGING);
     await libraryPage.addSelectedExplorationToPlaylist();
     await learnerDashboardPage.get();
-    await learnerDashboardPage.navigateToPlayLaterExplorationSection();
+    await learnerDashboardPage.navigateToCommunityLessonsSection();
     await learnerDashboardPage.expectTitleOfExplorationSummaryTileToMatch(
       EXPLORATION_SINGING);
     await users.logout();
   });
 
+  it('should display correct topics in edit goals, suggested for you section,' +
+    ' current goals and continue where you left off section', async function() {
+    var TOPIC_NAME = 'Learner Dashboard Topic 1';
+    var TOPIC_URL_FRAGMENT_NAME = 'ld-topic-one';
+    var TOPIC_DESCRIPTION = 'Topic description';
+    await users.createAndLoginCurriculumAdminUser(
+      'creator@learnerDashboard1.com', 'learnerDashboard1');
+    var handle = await browser.getWindowHandle();
+    await learnerDashboardPage.get();
+    await learnerDashboardPage.navigateToHomeSection();
+    await learnerDashboardPage.expectNumberOfTopicsInSuggestedForYou(0);
+    await learnerDashboardPage.navigateToProgressSection();
+    await learnerDashboardPage.expectNumberOfStoriesInCompletedStory(0);
+    await topicsAndSkillsDashboardPage.get();
```

Comment:
Can we check this expectation await topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(0) after this.

Reply:
Done.